addons/sitting/functions/fnc_getRandomAnimation.sqf

```diff
@@ -46,4 +46,4 @@ _animations = [
 ];
 
 // Select random animation
-(_animations select (floor (random (count _animations))))
+(_animations select floor random count _animations)
```

Comment:
I am not certain if this is improving the readability - it doesn't matter for performance so I'd suggest just leaving those in.